tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/test_sms_report.py

```diff
@@ -0,0 +1,81 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+from marionette_driver import expected, By, Wait
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.base import Base
+from gaiatest.apps.messages.app import Messages
+
+
+class TestSmsReport(GaiaTestCase):
+
+    def test_sms_report(self):
+        """This test sends a text message to itself. It waits for a reply message.
+           Then it checks the info in the report.
+        """
+        _text_type = (By.CSS_SELECTOR, 'p[data-l10n-id="message-type-sms"]')
+        _sender = (By.CSS_SELECTOR, 'a[data-name]')
+        _receiver = (By.CSS_SELECTOR, 'a.suggestion.js-contact-info')
+        
+        # Configuring the test
+        time_sent = time.time()
+        _text_message_content = "Automated Test %s" % str(time_sent)
+        _test_phone_number = self.testvars['phone_numbers']
+        
+        error_range_sent_in_seconds = 1200
+        error_range_rec_in_seconds = 3000
+        
+        # Sending the message
+        self.messages = Messages(self.marionette)
+        self.messages.launch()
+
+        new_message = self.messages.create_new_message(recipients=_test_phone_number,
+                                                       message=_text_message_content)
+        self.message_thread = new_message.tap_send()
+        self.message_thread.wait_for_received_messages()
+
+        last_received_message = self.message_thread.received_messages[-1]
+        last_message = self.message_thread.all_messages[-1]
+        
+        # Long Tap on received message and check the report: SMS sent received and time
+        report = last_received_message.open_report()
+
+        # Checking that the data in the received report is correct
+        # Type of message
+        self.assertTrue(report.is_message_an_sms)
+
+        # From, sender is the right one
+        self.assertEqual(_test_phone_number, report.sender)
+
+        # Sent date
+        time_sent_err = (time_sent + error_range_sent_in_seconds) 
+        time_sent_err_str = str(time_sent_err) 
+        self.assertLessEqual(str(report.is_date_sent_ok),time_sent_err_str)
+        
+        # Received date
+        time_rec_err = (time_sent + error_range_rec_in_seconds)
+        time_rec_err_str = str(time_rec_err)
+        self.assertLessEqual(str(report.is_date_rec_ok), time_rec_err_str)  
+
+        report.close_report()
+
+        # Long Tap on sent message and check the report: SMS sent and time (long_press)
```

Comment:
This comment is out to date. The test doesn't show that we have to long press the message to go to the report, anymore. That could be okay today because that's what actually happens.
However, if one day the way to access the report changes, we'll have to change the open_report() method and update this comment. As it's located in another file, there's some good chance the person who'll fix it, will forget to update that comment. So it might mislead the next reader. 

So like the comments that duplicate the lines below, I'd remove it. It looks safer.